File: pre-contest/a_billeterie/solutions/a_billeterie.php
<?php

fscanf(STDIN, "%d", $prix2nde);
fscanf(STDIN, "%d", $prix1ere);
fscanf(STDIN, "%d", $nbBillets2nde);
fscanf(STDIN, "%d", $nbBillets1ere);

echo ($prix2nde * $nbBillets2nde) + ($prix1ere * $nbBillets1ere)  . "\n";
?>

File: pre-contest/a_billeterie/solutions/a_billeterie.cs
using System;

namespace c_sharp
{
    internal class Program
    {
        public static void Main(string[] args)
        {
            var prix2nde = int.Parse(Console.ReadLine());
            var prix1ere = int.Parse(Console.ReadLine());
            var nbBillets2nde = int.Parse(Console.ReadLine());
            var nbBillets1ere = int.Parse(Console.ReadLine());
            
            Console.WriteLine((prix2nde * nbBillets2nde) + (prix1ere * nbBillets1ere));
        }
    }
}
